src/python/Utils/ProcFS.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+File       : ProcFS.py
+Author     : Valentin Kuznetsov <vkuznet AT gmail dot com>
+Description: This module provides function which rely on Linux proc fs
+"""
+# system modules
+import os
+
+# 3rd party modules
+import psutil
```

Comment:
I fear that importing this module in the worker node might hit an import error exception (as we do not have psutils in the grid nodes). So you might want to add a try/except here.

Reply:
Added try/except and fallback to use `processStatusViaProcFS`. But obvious question here is why do we need psutil implementation if on worker node we don't have it. The initial implementation via procfs is free from this issue.

Reply:
I know that is not ideal, but if the try/except properly handle the events where `psutils` cannot be loaded, we are fine with that. Sorry if this point was missed previously, eventually requiring additional dev effort